### PR TITLE
Adapt Docker Proxy

### PR DESCRIPTION
**Pull request by michal-dagan**
## Related Issues
fixes:https://jira-dc.paloaltonetworks.com/browse/CIAC-8399

## Description
https://confluence-dc.paloaltonetworks.com/pages/viewpage.action?pageId=121555211

### CODE REVIEW

File: demisto_sdk/commands/common/docker_helper.py [hunk 1]
```diff
@@ -329,8 +328,8 @@ def create_image(
 
     @staticmethod
     def get_image_registry(image: str) -> str:
-        if os.getenv("CONTENT_GITLAB_CI") and "code.pan.run" not in image:
-            return f"docker-io.art.code.pan.run/{image}"
+        if os.getenv("CONTENT_GITLAB_CI") and "code.pan.run" not in image:  # todo
```
Comment (michal-dagan) [inline demisto_sdk/commands/common/docker_helper.py:331]:
```suggestion
        if os.getenv("CONTENT_GITLAB_CI") and DOCKER_IO not in image:
```

File: demisto_sdk/commands/common/constants.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "CI_PROJECT_ID", "1061"
 )  # Default value is the ID of the content repo on GitLab
 ENV_SDK_WORKING_OFFLINE = "DEMISTO_SDK_OFFLINE_ENV"
+DOCKER_REGISTRY_URL = os.getenv("DOCKER_REGISTRY_URL", "docker-io.art.code.pan.run")
```
Comment (michal-dagan) [inline demisto_sdk/commands/common/constants.py:27]:
```suggestion
DOCKER_REGISTRY_URL = os.getenv("DOCKER_IO", "docker-io.art.code.pan.run")
```

File: demisto_sdk/commands/common/docker_helper.py [hunk 3]
```diff
@@ -75,7 +78,7 @@ def init_global_docker_client(timeout: int = 60, log_prompt: str = ""):
                 DOCKER_CLIENT.login(
                     username=docker_user,
                     password=docker_pass,
-                    registry="https://index.docker.io/v1",
+                    registry=registry,
```
Comment (michal-dagan) [inline demisto_sdk/commands/common/docker_helper.py:81]:
```suggestion
                    registry="https://index.docker.io/v1",
```

File: demisto_sdk/commands/common/docker_helper.py [hunk 4]
```diff
@@ -95,6 +97,7 @@ def docker_login(docker_client) -> bool:
     Returns:
         bool: True if logged in successfully.
     """
+
```
Comment (michal-dagan) [inline demisto_sdk/commands/common/docker_helper.py:100]:
```suggestion
```

File: demisto_sdk/commands/common/constants.py [hunk 5]
```diff
@@ -24,6 +24,7 @@
     "CI_PROJECT_ID", "1061"
 )  # Default value is the ID of the content repo on GitLab
 ENV_SDK_WORKING_OFFLINE = "DEMISTO_SDK_OFFLINE_ENV"
+DOCKER_REGISTRY_URL = os.getenv("DOCKER_IO", "docker-io.art.code.pan.run")
```
Comment (DeanArbel) [inline demisto_sdk/commands/common/constants.py:27]: Change to `docker.io`

**Review by DeanArbel — APPROVED** [review]

File: demisto_sdk/commands/common/constants.py (hunk 5 above)
Comment (michal-dagan) [inline demisto_sdk/commands/common/constants.py:27]:
```suggestion
DOCKER_REGISTRY_URL = os.getenv("DOCKER_IO", "docker.io")
```

**Review by ilaner — APPROVED** [review]
nice!

**Pull request merged**